layouts/partials/fragments/member.html

```diff
@@ -37,6 +38,11 @@ <h5>
           </div>
         </div>
       {{ end }}
+      {{- if eq $members 0 -}}
+        <div class="text-danger alert-danger text-center">
+          {{- printf "Currently there is no configured member content files present. To add new members add a content file within index/%s" $path -}}
```

Comment:
Please change the `index/%s` to something that would be correct in other pages as well.